extendeddatasquare.go

```diff
@@ -232,3 +259,23 @@ func deepCopy(original [][]byte) [][]byte {
 func (eds *ExtendedDataSquare) Width() uint {
 	return eds.width
 }
+
+// validateEdsWidth returns an error if edsWidth is not a valid width for an
+// extended data square.
+func validateEdsWidth(edsWidth uint) error {
+	if edsWidth%2 != 0 {
+		return errors.New("square width must be even")
```

Comment:
[Suggestion] We can also report the `edsWidth` in the error for additional context.

Reply:
I'd really like to avoid pushing a commit and dismissing existing approvals b/c there are PRs stacked on top of this. Will tackle in a follow up issue.